ldi-nifi/ldi-nifi-processors/ldes-client-processor/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldi/processors/config/LdesProcessorProperties.java

```diff
@@ -228,8 +230,14 @@ private LdesProcessorProperties() {
 			.addValidator(StandardValidators.URI_VALIDATOR)
 			.build();
 
-	public static String getDataSourceUrl(final ProcessContext context) {
-		return context.getProperty(DATA_SOURCE_URL).getValue();
+	public static List<String> getDataSourceUrl(final ProcessContext context) {
+		var urls = Arrays.stream(context.getProperty(DATA_SOURCE_URLS).getValue().split(",")).toList();
```

Comment:
This is a detail and I am fine if you ignore it but since you already have a stream of strings, might as well add a `.map(String::trim)` to help out people that put spaces after their comma's in the config :)